live-examples/js-examples/string/string-substr.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var str = 'Mozilla';
+
+console.log(str.substr(1, 2));
+// expected output: 'oz'
+
+console.log(str.substring(2));
```

Comment:
This is still `substring` not `substr` :).

Reply:
fixed